### PR TITLE
Support and fixes for building on Windows.

### PR DESCRIPTION
**Pull request by cristianberneanu**

### CODE REVIEW

**Comment by edongashi** [comment]
Sigh, what a headache!

File: src/pg_diffix.c [hunk 1]
```diff
@@ -11,14 +11,19 @@
...
 
 #include <limits.h>
+
+#ifdef _WIN64
+#define __WORDSIZE 64
+#endif
+
 #if __WORDSIZE != 64
 #error "This module requires a 64-bit target architecture!"
 #endif
```
Comment (edongashi) [inline src/pg_diffix.c:21]: Looks like we can't catch a 32 bit build on Windows. But it's pretty much extinct anyway...

**Review by edongashi — APPROVED** [review]

File: src/pg_diffix.c (hunk 1 above)
Comment (cristianberneanu) [reply src/pg_diffix.c:21]: We do catch it, on a 32-bit build `__WORDSIZE` would remain undefined, so the error will be raised and the build will be stopped.

Comment (edongashi) [reply src/pg_diffix.c:21]: Ah, now I see that it's `#ifdef _WIN64`.

**Pull request merged**